tools/acrntrace/README.rst

```diff
@@ -12,6 +12,67 @@ A ``scripts`` directory includes scripts to analyze the trace data.
 Usage
 *****
 
+The ``acrntrace`` tool runs on the Service OS (SOS) to capture trace data and
+output to trace file under ``/tmp/acrntrace`` with raw (binary) data format.
+
+Options:
+
+-h			print this message
+-t period		specify polling interval in milliseconds [1-999]
+-c			clear the buffered old data
+
+The ``acrntrace_format.py`` is a offline tool for parsing trace data (as output
+by acrntrace) to human-readable formats based on given format.
+
+Here's an explanation of the tool's parameters:
+
+.. code-block:: none
+
+   acrntrace_format.py [options] [formats] [trace_data]
+
+Options:
+
+-h    print this message
+
+- A *formats* file specifies the rules to reformat the *trace_data* collected by
```

Comment:
The option list input format is picky about spaces and alignment:  no space between the ``-`` and the option name and at least two spaces after the option name and the text that explains the option:

    -A formats_file     specifies the rules to reformat the *trace_data*
                        collected by ``acrntrace`` into a human-readable 
                        text form. 

    The rules in this *formats_file* are of the form::

       event_id  text_format_string

    The text_format_string may include ...

Reply:
@dbkinder Sorry not make it clear. There is no option "-A" in acrntrace_format.py.  Will remove the confusing "-A" prefix in next version.